src/world/area_mac/mac_01/npc/flower_gate.inc.c

```diff
@@ -331,10 +339,14 @@ s32 N(ItemList_MagicalSeeds)[] = {
 EvtScript N(EVS_MinhT_PlantSeed) = {
     EVT_CALL(SetNpcFlagBits, NPC_SELF, NPC_FLAG_IGNORE_PLAYER_COLLISION, TRUE)
     EVT_CALL(SpeakToPlayer, NPC_MinhT, ANIM_MinhT_Talk, ANIM_MinhT_Idle, 0, LVar4)
-    EVT_WAIT(10)
+    EVT_WAIT(10 * DT)
     EVT_CALL(UseSettingsFrom, CAM_DEFAULT, 266, 27, 373)
     EVT_CALL(SetPanTarget, CAM_DEFAULT, 266, 27, 373)
+#if VERSION_PAL
```

Comment:
This pattern is appearing quite a lot, even for this single area. Would be nicer if it could be a single macro `FOO(1, / DT)` (though starting with an operator looks a bit weird)

Reply:
maybe EVT_TIMESCALED(1) that is an int on ntsc but a float divided by DT on pal

unsure about the name

Reply:
There's also cases where it's multiplied by DT instead of divided (https://github.com/pmret/papermario/pull/967/files#diff-6fe3ae3603c232cbade9a79373aad40a5a940e12d115bcc449e5c932e6b5ed11R316).